.github/workflows/sandbox.yaml

```diff
@@ -0,0 +1,51 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+name: Sandbox
+jobs:
+  check:
+    name: Sandbox
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+      - id: get-release
+        name: get-latest-release
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: lf-lang/lingua-franca
+      - name: Integration Test
+        run: |
+            # uninstalling java11 and installing java17
+            sudo apt remove openjdk-11-jre-headless
```

Comment:
The first half of these commands can be replaced by:
```
 - uses: actions/setup-java@v3
      with:
        distribution: temurin
        java-version: 17
```

Reply:
done